docs/INSTALLATION.rst

```diff
@@ -42,3 +42,9 @@ Installation instructions:
 
     Or from the root directory of the unpacked archive.
     $ pip install .
+
```

Comment:
Note, none of the above install instructions work anymore, unless you explicitly do the below instruction. This is related to secure-systems-lab/securesystemslib#153

Reply:
The TUF client should be able to perform an update without needing the optional SSLib deps, provided it only needs to verify Ed25519 keys.  Correct me if I'm wrong.

I should probably update the quickstart and CLI guides to make it clear that the user must install securesystemslib[crypto,pynacl] to generate keys, metadata, etc

Reply:
@SantiagoTorres Let me know if https://github.com/theupdateframework/tuf/pull/756 helps and can be merged.